scripts/flamingfir-deploy.sh

```diff
@@ -1,16 +1,26 @@
 #!/bin/bash
 
-TIMEOUT=60
+RETRY_COUNT=10
+RETRY_ATTEMPT=0
+SLEEP_TIME=15
 TARGET_HOST="$1"
-COMMIT=$(echo ${CI_BUILD_REF} | cut -c -9)
-DOWNLOAD_URL="https://releases.parity.io/substrate/x86_64-debian:stretch/2.0.0-${COMMIT}/substrate"
+COMMIT=$(cat artifacts/VERSION | cut -d- -f2)
```

Comment:
https://github.com/paritytech/substrate/blob/master/.gitlab-ci.yml#L230
I haven't checked but I assume the path must include the whole VERSION file content.